layouts/shortcodes/site-searchbar.html

```diff
@@ -1,3 +1,3 @@
-<div class="col-sm-6 col-md-6 col-lg-6 mx-auto py-3">
-{{partial "search-input" .}}
-</div>
\ No newline at end of file
+<section class="k8s-search">
+{{- partial "search-form.html" . }}
+</section>
```

Comment:
Making this a section by itself allows us to make the search form take up less than the full viewport width.